Obsidian/Net/Rcon/RconConnection.cs

```diff
@@ -244,6 +244,13 @@ await SendAsync(new RconPacket(currentEncoding)
 
                         try
                         {
+                            logger.LogInformation("Executing '{Prefix}{Command}'", CommandHandler.DefaultPrefix, packet.PayloadText);
+
+                            server.Operators.GetOnlineOperators().ForEach(op =>
+                            {
+                                op.SendMessageAsync($"[RCON: {CommandHandler.DefaultPrefix}{packet.PayloadText}]", MessageType.System);
```

Comment:
We should have a more public/general method to broadcast command execution messages instead of only putting it into RconConnection.cs

Reply:
I don't see any other use case for sending something to all operators, also it would've been possible to refactor this at a later point if it's needed.

Reply:
Notchian servers inform an operator about the output from commands executed by other operators

Reply:
Sorry to inform you, but RCON also [as stated here](https://minecraft.fandom.com/wiki/Server.properties#Java_Edition_2:~:text=online%20operators.-,broadcast%2Drcon%2Dto%2Dops,Send%20rcon%20console%20command%20outputs%20to%20all%20online%20operators.,-difficulty).

Also sending just the command output is not a sensible option until the rewrite/rework of the command system. RCON just sending `help` would print the help to all OPs.